scripts/karma.coffee

```diff
@@ -151,6 +164,10 @@ module.exports = (robot) ->
     if (karma.getAllowance(user) > 0) and (allow_self is true or user.name.toLowerCase() != subject)
       karma.increment subject, user
       msg.send "#{subject} #{karma.incrementResponse()} (Karma: #{karma.get(subject)})"
+      if (msg.robot.name == "robort")
+        robot.messageRoom("karma", karma.notify(msg, subject, user, "incremented"));
```

Comment:
This should probably be a property instead of hard coded to the "karma" room. That would also get rid of the need for this if/else.